backend/Match/views.py

```diff
@@ -27,10 +27,10 @@ def get(self, request):
         if token.user_left_side.id != request.user.id:
             return Response({'error': 'You are not the host user in the token'}, status=status.HTTP_403_FORBIDDEN)
 
-        match, player_left, player_right = GameSetupManager.create_match_and_its_players(token)
+        match, player_left, player_right = MatchSetupManager.create_match_and_its_players(token)
 
         if not all([match, player_left, player_right]):
-            return Response({'error': 'Something went wrong when creating the match and players'}, status=status.HTTP_500_INTERNAL_SERVER_ERROR)
+            return Response({'error': 'Something went wrong when creating the match and players'}, status=status.HTTP_403_FORBIDDEN)
```

Comment:
Technically this is not really it xD
Maybe 422?

Reply:
Correction, maybe as in the Error 500 thread mentioned. We should find the right 5xx code. My mistake for misremembering the eval form

Reply:
In school it would likely be 507, I'd wager (insufficient storage), but we should be confident of the cause in the different situations,  I'd say

Reply:
we can talk about this more as a group later!